### PR TITLE
[TW-419] Document Azure API Management

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Added "Deploying to Azure API Management" page
- Updated "Deploying an API overview" page
- Updated screenshots

### CODE REVIEW

**Review by HeatherStenson — APPROVED** [review]
Looks great!

**Review by vinitshahdeo — APPROVED** [review]

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+1. Select **APIs** in the sidebar and select an API.
+1. Under **Power up your API**, select **Deployments**, and then select **Azure API Management**.
+1. Select the service you want to export to, and then select **Export API**.
+1. Select the branch or version to export.
+1. Enter a brief description for the export.
+1. (Optional) If you want to export the schema to an existing revision, select the check box. Select the **Revision ID** of the revision you want to update.
+
+    > If you don't select the checkbox, Postman exports the API to a new revision.
+
+1. Select **Export**.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:110]: This screenshot needs to be changed - see TW-645

**Review by vinitshahdeo — APPROVED** [review]

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:107]: Updated screenshot

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+    * **Subscription** - The [subscription](https://docs.microsoft.com/en-us/azure/api-management/api-management-subscriptions) where the service was created.
+    * **Resource Group** - The [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal) that contains the service.
+    * **Service** - The Azure API Management [service instance](https://docs.microsoft.com/en-us/azure/api-management/api-management-key-concepts) used for deploying your API.
+    * **Azure API Version** - The Azure API [version](https://docs.microsoft.com/en-us/azure/api-management/api-management-versions) that corresponds to your API in Postman.
+
+1. To connect more Azure API Management services, select **+ Add Service**. To remove a service, select **Remove**.
+1. When you're finished adding services, select **Connect**.
+
+    <img alt="Azure API Management setup" src="https://assets.postman.com/postman-docs/v10/azure-apim-setup-v10.jpg" width="513"/>
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:42]:
```suggestion
    <img alt="Azure API Management setup" src="https://assets.postman.com/postman-docs/v10/azure-apim-setup-v10.jpg" width="513px"/>
```

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+1. Select **Deployments**, and then select **Import Definition**.
+1. Select the **Definition type** (OpenAPI 2.0 or OpenAPI 3.0).
+1. Select the **Azure service** to import the definition from.
+1. Select **Import**.
+
+<img alt="Importing a definition" src="https://assets.postman.com/postman-docs/v10/azure-apim-import-schema-v10.jpg" width="651px"/>
+
+## Exporting your API to Azure API Management
+
+Exporting an OpenAPI definition makes it available in the connected Azure API Management service. OpenAPI 2.0 and 3.0.x definition are supported. You can export the definition to an existing revision or to a new revision.
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:96]:
```suggestion
Exporting an OpenAPI definition makes it available in the connected Azure API Management service. OpenAPI 2.0 and 3.0.x definitions are supported. You can export the definition to an existing revision or to a new revision.
```

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md [hunk 4]
```diff
@@ -31,15 +31,20 @@ Stay on top of your API deployments in Postman by connecting to an API gateway.
...
 
 * **Amazon API Gateway** - View the active deployment for each stage, recent exports from Postman, and details about each deployment. You can also export or deploy your HTTP API definition from Postman to your gateway. Learn more about [Viewing Amazon API gateway deployments](/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-aws/#viewing-amazon-api-gateway-deployments).
 * **Apigee X** - View details about your Apigee API proxy and proxy endpoints. You can also view details about your Apigee environments. Learn more about [Viewing Apigee X deployments](/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-apigee/#viewing-apigee-x-deployments).
+* **Azure API Management** View deployments for each of your Azure API Management services. You can view the revision history, changelog, and export history. You can also export your API definition from Postman to Azure API Management. Learn more about [Viewing Azure API deployments](/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure/#viewing-azure-api-deployments).
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md:50]:
```suggestion
* **Azure API Management** - View deployments for each of your Azure API Management services. You can view the revision history, changelog, and export history. You can also export your API definition from Postman to Azure API Management. Learn more about [Viewing Azure API deployments](/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure/#viewing-azure-api-deployments).
```

**Review by HeatherStenson — APPROVED** [review]
Looks good!

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:96]: And this is why I ask you to review things 😅

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md (hunk 4 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-overview.md:50]: Another good catch! 😸

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+1. Select **Deployments**, and then select **Import Definition**.
+1. Select the **Definition type** (OpenAPI 2.0 or OpenAPI 3.0).
+1. Select the **Azure service** to import the definition from.
+1. Select **Import**.
+
+<img alt="Importing a definition" src="https://assets.postman.com/postman-docs/v10/azure-apim-import-schema-v10.jpg" width="651px"/>
+
+## Exporting your API to Azure API Management
+
+Exporting an OpenAPI definition makes it available in the connected Azure API Management service. OpenAPI 2.0 and 3.0.x definitions are supported. You can export the definition to an existing revision or to a new revision.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:96]: Is this OpenAPI 3.0.x and not 3.1? (If so, this is fine, but wasn't sure if that should be 3.x)

**Review by jkonrath-postman — APPROVED** [review]
One little question, but otherwise LGTM

File: src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md (hunk 5 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/deploying-an-api/deploying-an-api-azure.md:96]: I think we can change this to just OpenAPI 3.0 which is what we state elsewhere in this topic.

@nishantpostman does that sound right?

**Review by shashankawasthi88 — COMMENTED** [review]
Mark this looks good but we should also add the ability to start from the import modal to import the API from Azure APIM to start creating an API

**Comment by mark-anthony-postman** [comment]
> Mark this looks good but we should also add the ability to start from the import modal to import the API from Azure APIM to start creating an API

@shashankawasthi88 Importing from Azure gateway to create a new API is covered on the "Importing an API" page. See this PR: https://github.com/postmanlabs/postman-docs/pull/4242

**Pull request merged**